File: Blind75/Binary Tree Equal/Binary_Tree_Equal.js
function binaryTreeEqual(a, b){
  if (a == null && b == null) return true;
  if (a == null || b == null) return false;
  if (a.val != b.val) return false;

  return binaryTreeEqual(a.left, b.left) && binaryTreeEqual(a.right, b.right);
}

File: Blind75/Binary Tree Equal/Binary_Tree_Equal.py
def binaryTreeEqual(a, b):
    if a is None and b is None: return True 
    if a is None or b is None: return False
    
    if a.val != b.val: return False
    return binaryTreeEqual(a.left, b.left) and binaryTreeEqual(a.right, b.right)